alicloud/resource_alicloud_network_interface_test.go

```diff
@@ -0,0 +1,544 @@
+package alicloud
+
+import (
+	"fmt"
+	"github.com/aliyun/alibaba-cloud-sdk-go/sdk/requests"
+	"log"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aliyun/alibaba-cloud-sdk-go/services/ecs"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-alicloud/alicloud/connectivity"
+)
+
+func init() {
+	resource.AddTestSweepers("alicloud_network_interface", &resource.Sweeper{
```

Comment:
I think this resource sweeper test should be dependence of vswitch, security group